src/classes/Batchs/MinAccountOppBatch.cls

```diff
@@ -0,0 +1,64 @@
+/**
+ * Created by Mariya on 17.01.2023.
+ */
+
+public with sharing class MinAccountOppBatch implements Database.Batchable<sObject>, Schedulable, Database.Stateful {
+
+    public Iterable<sObject> start(Database.BatchableContext param1) {
+        return Database.getQueryLocator('SELECT Id FROM Account');
+    }
+
+    public void execute(Database.BatchableContext param1, List<Account> accounts) {
+
+        Logger logger = new Logger();
+
+        try {
+            List<Opportunity> opportunitiesToSum = new List<Opportunity>();
+
+            for (Account acc : accounts) {
+                List<Opportunity> opportunities = [SELECT Id, Amount FROM Opportunity WHERE AccountId = :acc.Id];
```

Comment:
CRITICAL
Avoid DML and SOQL inside for loop as it will fail by limit of 101 query:
https://www.apexhours.com/apex-code-best-practices/